lhotse/features/io.py

```diff
@@ -0,0 +1,177 @@
+from abc import ABCMeta, abstractmethod
+from functools import lru_cache
+from pathlib import Path
+from typing import Type
+
+import lilcom
+import numpy as np
```

Comment:
hard for me to comment on this file without documentation...

Reply:
The docs are here now 🔥